packages/webapp/src/pages/Logs/Search.tsx

```diff
@@ -213,15 +303,32 @@ export const LogsSearch: React.FC = () => {
                 <Table.Body>
                     {table.getRowModel().rows?.length ? (
                         table.getRowModel().rows.map((row) => <OperationRow key={row.id} row={row} />)
-                    ) : (
+                    ) : operations.length <= 0 && !loading && readyToDisplay ? (
                         <Table.Row>
                             <Table.Cell colSpan={columns.length} className="h-24 text-center">
                                 No results.
                             </Table.Cell>
                         </Table.Row>
+                    ) : (
+                        <Table.Row>
+                            {table.getAllColumns().map((col, i) => {
+                                return (
+                                    <Table.Cell key={i}>
+                                        <Skeleton style={{ width: col.getSize() }} />
+                                    </Table.Cell>
+                                );
+                            })}
+                        </Table.Row>
                     )}
                 </Table.Body>
             </Table.Table>
+            {data && data.pagination.total > 0 && data.data.length > 0 && cursor.current && readyToDisplay && (
+                <div ref={bottomScrollRef}>
+                    <Button disabled={loading} variant="active" className="w-full justify-center" onClick={() => loadMore()}>
+                        {loading && <Spinner size={1} />} Load More
```

Comment:
Shouldn't it say "Loading more..." or just "Loading"

Reply:
good point, I'll put loading...